packages/ui/components/combobox/src/validators.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable max-classes-per-file */
+import { Validator } from '@lion/ui/form-core.js';
+
+export class IsMatchingAnOption extends Validator {
+  static get validatorName() {
+    return 'IsMatchingAnOption';
+  }
+
+  /**
+   * @param {unknown} [value]
+   * @param {string | undefined} [options]
+   * @param {{ node: any }} [config]
+   */
+  // eslint-disable-next-line class-methods-use-this
+  execute(value, options, config) {
+    const hasMatchingOptions =
```

Comment:
What does this return? `aria-setsize` seems to be available on every option: https://github.com/ing-bank/lion/blob/fd09f652d450580d7ad43d2c4466cb4ed42d30f0/packages/ui/components/listbox/src/ListboxMixin.js#L455

`hasMatchingOptions` will then always be false (because .length > 0): 
- the boolean returned (indicating whether feedback should be shown) will thus be `false`
- this indicates that there is never an error, even if the textbox value is not found...

Maybe I'm wrong btw, but it would be good to (dis)prove it with a test :)

Reply:
I think `hasMatchingOption(s)` can be determined with `this.checkedIndex !== -1`.
And then it might be good to think about the multipleChoice case as well

Reply:
Actually it worked, i got confused by my own naming :)